engine/src/main/kotlin/world/gregs/voidps/engine/data/definition/AmmoDefinitions.kt

```diff
@@ -9,14 +9,37 @@ import world.gregs.voidps.engine.timedLoad
 import world.gregs.yaml.Yaml
 
 /**
- * List of [AmmoDefinition.items] each weapons ammo_group can use ignoring
- * skill requirements and whether inside dungeoneering.
+ * The `AmmoDefinitions` class is responsible for decoding and managing a collection of ammo definitions.
+ * It inherits from `DefinitionsDecoder` with a specific type of `AmmoDefinition`.
```

Comment:
Loses all the useful information of the existing comment and isn't using [Kotlin doc syntax](https://kotlinlang.org/docs/kotlin-doc.html)